styleguide/assets/server.css

```diff
@@ -4944,9 +4931,16 @@ kbd {
   border-left: 3px solid var(--color-success);
 }
 
+/* dark theme overrides for Nextcloud 24 and earlier */
+.theme--dark .toastify.dialogs .toast-close {
+  /* close icon style */
+}
+.theme--dark .toastify.dialogs .toast-close.toast-close::before {
+  background-image: url("./close-dark.svg");
```

Comment:
Well, that shows an issue xD
https://github.com/nextcloud/nextcloud-dialogs/tree/master/styles

it should be encoded, it doesn't exists there :thinking: .

Reply:
https://github.com/nextcloud/nextcloud-dialogs/pull/691